src/components/AuthLogin/AuthLogin.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import Image from "next/image";
+
+import BreakLine from "./BreakLine";
+import EmailLogin from "./EmailLogin";
+import GoogleSignIn from "./GoogleSignIn";
+import UabcHeaderText from "./UabcHeaderText";
+import UabcLogo from "./UabcLogo";
+
+const AuthLogin = () => {
+  return (
+    <div className="dark h-dvh w-dvw bg-background grid place-items-center">
+      <div className="w-dvw px-4 flex-col h-full flex items-center justify-between py-8">
```

Comment:
can make this into one div

<div className="dark min-h-dvh w-dvw bg-background px-4 flex-col flex items-center justify-between py-8">